package.json

```diff
@@ -55,6 +55,7 @@
     "@types/d3-array": "^2.5.1",
     "@types/d3-axis": "^2.0.0",
     "@types/d3-scale": "^3.2.1",
+    "@types/d3-scale-chromatic": "^2.0.0",
```

Comment:
this is not used here but if you plan to use it later we can keep it

Reply:
this is used in `Plot.tsx:3`, I don't know why I didn't add it before